contracts/consensus/Block.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "./RLP.sol";
+
+/**
+ * @title Block provides functionality to work with blocks.
```

Comment:
this file should not be under `/consensus` rather under `/block` for now.  We can group contracts better later, for now let s not assume groupings

Reply:
Done.